Src/Program.cs

```diff
@@ -148,7 +148,15 @@ private void RunInstall(string extensionPath)
                 var installableExtension = _extensionManager.CreateInstallableExtension(extensionPath);
                 var identifier = installableExtension.Header.Identifier;
                 UninstallSilent(identifier);
-                _extensionManager.Install(installableExtension, perMachine: false);
+
+                var perMachine = false;
+                if(installableExtension.Header.AllUsers != perMachine)
```

Comment:
Gets me every time! I think its because I'm used to the auto-formatting. Because the } is automatically inserted, I don't get to type it and the formatting never happens. Or maybe its to do with the ). Feels like something has changed anyway. ;-)